zerver/lib/presence.py

```diff
@@ -156,7 +156,7 @@ def get_status_dict_by_realm(
         "status",
         "timestamp",
         "user_profile__email",
-        "user_profile__id",
+        "user_profile_id",
```

Comment:
I'm wondering about these - `user_profile__email` is being fetched, so perhaps it makes sense to have `user_profile__id`? Not sure, it seems fine to change as well.

Reply:
Yeah, I think this is still better, since in part it emphasizes that if/when we remove email addresses from this structure (which we will), it'll save a join.